qemu/tests/cfg/check_block_size.cfg

```diff
@@ -88,3 +88,18 @@
             nic_mode = tap
             physical_block_size_stg = 512
             logical_block_size_stg = 512
+        - 4096_4096_cluster_install:
+            no base
+            only s390x ovmf
```

Comment:
Will not consider other platforms? BTW, I think it would be better to disable default_bios on x86 in L30, but not here. https://github.com/autotest/tp-qemu/blob/c0d8bd805478e4d3568861577dcde6d7d415720c/qemu/tests/cfg/check_block_size.cfg#L29-L30

Reply:
I think this is because other archs would not run this test with default bios as well

Reply:
In practice this restriction means that this is the case for s390 only.
I think actually maybe you want to do that?

```
s390x:
    only ovmf
```

Reply:
I was meant to run this only on both s390x and ovmf, just updated. Please correct me if I'm wrong

Reply:
Actually, if you don't know the support status on each platform, you should not limit them according to your thinking. For x86, only ovmf supports it, and aarch64 use aavmf, they are both a part of EDK2. So instead of limiting them directly, I recommend removing it and disabling it in the future if other platforms really don't support it.

Reply:
you are right, will involve all feature owner on all arches to help review if they supported this or not